networks.json

```
@@ -123,6 +147,10 @@
     "42161": {
       "address": "0x6cf1e9ca41f7611def408122793c358a3d11e5a5",
       "transactionHash": "0x1246e1018edd2963aec95714c96beccc5c99af6ea38a43af3795a06624d60aa6"
+    },
+    "232": {
+      "address": "0x6cF1e9cA41f7611dEf408122793c358a3d11E5a5",
+      "transactionHash": "0x2b4f8b7e2f8b3c6d4f7e9b8d1a5e2c3f4b6d8e9f7a8b9c0d1e2f3a4b5c6d7e8f"
```

This appears to be incorrect, see https://explorer.lens.xyz/address/0x6cF1e9cA41f7611dEf408122793c358a3d11E5a5 .

```suggestion
      "transactionHash": "0xda19203f82d31db7f547db1faf71aff43ab1c569bea2acaf6f611df77a28900d"
```